app/lib/efile/az/az321_calculator.rb

```diff
@@ -0,0 +1,68 @@
+class Efile::Az::Az321Calculator < ::Efile::TaxCalculator
+  attr_reader :lines, :value_access_tracker
+
+  def initialize(value_access_tracker:, lines:, intake:)
+    @value_access_tracker = value_access_tracker
+    @lines = lines
+    @intake = intake
+  end
+
+  def calculate
+    set_line(:AZ321_LINE_4H, :calculate_line_4h)
+    set_line(:AZ321_LINE_4, :calculate_line_4)
+    set_line(:AZ321_LINE_5, :calculate_line_5)
+    set_line(:AZ321_LINE_11, :calculate_line_11)
+    set_line(:AZ321_LINE_12, :calculate_line_12)
+    set_line(:AZ321_LINE_13, :calculate_line_13)
+    set_line(:AZ321_LINE_20, :calculate_line_20)
+    set_line(:AZ321_LINE_22, :calculate_line_22)
+  end
+
+  private
+
+  def calculate_line_4
+    # If you made contributions to more that three qualifying charitable organization,
+    # enter the amount from line 4h of the Continuation Sheet, otherwise enter "0"
+    line_or_zero(:AZ321_LINE_4H)
+  end
+
+  def calculate_line_4h
```

Comment:
very minor, but can you move this above `calculate_line_4` so it matches the order they're called in?